README.md

```diff
@@ -95,6 +95,12 @@ In order to run them, use this command line:
 
 ---
 
```

Comment:
please mention config values used in the code eg
```scala

  private val interval = config.getInt("notification.email.interval")
  private val toAddress = config.getString("notification.email.address")
  private val queueThreshold = config.getInt("notification.email.queueThreshold")
```

Reply:
Done.